### ISSUE TITLE
Highlight list

### ISSUE THREAD

**Issue by kiuKisas** [nkrkv/nvim-treesitter-rescript#2]
![20210825_18h19m01s_grim](https://user-images.githubusercontent.com/12698940/130827623-2440f2e7-5fad-4cdf-a267-394a9fd57733.png)

It could be great if:
- [ ] `list{...}` is highlight
- [ ] `others` in my screenshot is highlight

**Comment by nkrkv** [comment]
Yep. Lists are obviously missing. In this context, I think you are right, `others` should take the same coloring as `current`.
